CONTRIBUTING.md

```
@@ -5,24 +5,31 @@ Thank you for contributing to asusctl. This guide outlines project bounds, issue
 ## Project scope and support bounds
 
 ### Scope
-The goal of this project is making ASUS hardware usable under Linux.
-
-### Supported distributions
-Arch Linux and Arch-based distributions are officially supported. Issues specific to unsupported distributions like Ubuntu may be closed without investigation due to maintainer time constraints.
-
+The goal of this project is to bring full support to all features ASUS officially allows for the same hardware on Linux.
 ## Reporting issues
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)